src/vs/workbench/contrib/terminal/browser/terminalProcessExtHostProxy.ts

```diff
@@ -66,11 +64,6 @@ export class TerminalProcessExtHostProxy extends Disposable implements ITerminal
 		this._onProcessData.fire(data);
 	}
 
-	public emitTitle(title: string): void {
-		hasReceivedResponseFromRemoteExtHost = true;
```

Comment:
This PR runs on the assumption that this function is not actually in use atm and no longer required. I may be mistaken though as this variable was introduced in a fix

Reply:
When we improve support for `experimentalUseTitleEvent` we may want to keep this and move actually fire it as a process event instead. I'll add this back but keeping it unused seems reasonable.